### PR TITLE
[RFC][NI] New component for status tags or labels

### PR DESCRIPTION
**Pull request by fredcerdeira**
I think this should be a component because probably will be used in more areas of the different platforms.

Example of use: In the new Management-Enquires area of AP
https://projects.invisionapp.com/d/main/default/#/console/10009152/215179492/preview

ps: The new colors should be added to the uniq colors

### CODE REVIEW

**Comment by fredcerdeira** [comment]
Added new colors for use in the new status-tags. 
@mmelo review please

File: src/styles/_common/_colors.scss [hunk 1]
```diff
@@ -111,8 +111,14 @@ $palette: (
     'border': rgba(0, 60, 88, 0.3),
     'border-active': rgba(0, 60, 88, 0.7),
   ),
-  cookieAlert: (
+  'cookieAlert': (
```
Comment (asantos00) [inline src/styles/_common/_colors.scss]: Don't we use dashed names instead of camelCase ? (cookieAlert => cookie-alert)

Comment (asantos00) [inline src/styles/_common/_colors.scss]: At least that's what we do in other palette colors

Comment (asantos00) [inline src/styles/_common/_colors.scss]: And stumbled upon this PR, sorry

Comment (mmelo) [inline src/styles/_common/_colors.scss]: this ^

Comment (fredcerdeira) [inline src/styles/_common/_colors.scss]: @asantos00 cookieAlert was already camelCased but I agree that dashed should be standard.

File: src/styles/_components/_status-tags.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+   status-tag
+    - example: https://projects.invisionapp.com/d/main/default/#/console/10009152/215179492/preview : expired and pending status
+    - the new colors should be added to the uniq colors
+*/
+.status-tag {
+  @include font("IntervalNextMed");
+  border-radius: 2px;
+  font-size: em(14px);
+  padding: 2px 9px 4px 9px;
+  display: inline-block;
```
Comment (mmelo) [inline src/styles/_components/_status-tags.scss]: opt for `display: block;` instead as the placement of this component will be decided on a per-platform-basis.

File: src/styles/_common/_colors.scss (hunk 1 above)
Comment (mmelo) [inline src/styles/_common/_colors.scss]: @fredcerdeira exactly

File: src/styles/_components/_status-tags.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/*
+   status-tag
+    - example: https://projects.invisionapp.com/d/main/default/#/console/10009152/215179492/preview : expired and pending status
+    - the new colors should be added to the uniq colors
+*/
```
Comment (mmelo) [inline src/styles/_components/_status-tags.scss]: please remove the comment.

File: src/styles/_common/_colors.scss [hunk 4]
```diff
@@ -111,8 +111,14 @@ $palette: (
...
     'background': #e1e4e6
+  ),
+  'statusTag': (
```
Comment (asantos00) [inline src/styles/_common/_colors.scss]: btw, if you fix the other one, fix this too @fredcerdeira

**Comment by fredcerdeira** [comment]
Changes done. Review it again please when possible.

**Pull request merged**